package.json

```diff
@@ -33,6 +33,7 @@
     "@typescript-eslint/parser": "6.4.0",
     "danger": "11.2.8",
     "editorconfig-checker": "5.1.1",
+    "esbuild": "0.19.2",
```

Comment:
if you remove node_modules and reinstall them again this should work without this dependency, maybe we will add it later but only for frontend
```suggestion
```